package.json

```diff
@@ -11,7 +11,7 @@
     "compile": "lerna run compile",
     "clean": "lerna run --parallel clean",
     "clean-docs": "rm -rf docs/* && cp -rf docs-static/* docs/.",
-    "copy-docs-app": "mkdir -p docs/docs/v2 && cp -rf packages/docs-app/dist/* docs/docs/v2/.",
+    "copy-docs-app": "mkdir -p docs/docs/v3 && cp -rf packages/docs-app/dist/* docs/docs/v3/.",
```

Comment:
updating where `develop` docs assets get copied to. updating other versions of docs site is a more manual process.